ui/templates/browse.html

```diff
@@ -128,5 +128,6 @@ <h1>Φορτίο (fortio) v{{.Version}}{{.Extra}}</h1>
 fortio_load(files.value)
 </script>
 {{end}}
+<p>Go to <a href='./'>Top</a>.</p>
```

Comment:
it’s a bit subtle but there are really more than 1 top depending on report only vs full server so I left the label ambiguous rather than add more variables and mode